### PR TITLE
docs: Update CONTRIBUTING file

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.440.4511841640.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.440.4511841640.xml)  
[color_metro_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.440.4511841640.swift)  
[color_metro_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.440.4511841640.kt)  
[color_metro_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.440.4511841640.ts)  
[color_sberHealth_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.440.4511841640.swift)  
[color_sberHealth_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.440.4511841640.kt)  
[color_sberHealth_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.440.4511841640.ts)  
[color_sbermarket_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.440.4511841640.swift)  
[color_sbermarket_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.440.4511841640.kt)  
[color_sbermarket_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.440.4511841640.ts)  
[color_sberprime_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.440.4511841640.swift)  
[color_sberprime_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.440.4511841640.kt)  
[color_sberprime_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.440.4511841640.ts)  
[color_selgros_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.440.4511841640.swift)  
[color_selgros_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.440.4511841640.kt)  
[color_selgros_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.440.4511841640.ts)  
[color_smbusiness_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.440.4511841640.swift)  
[color_smbusiness_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.440.4511841640.kt)  
[color_smbusiness_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.440.4511841640.ts)  
[PlasmaTokensColor--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.440.4511841640.swift)  
[shadow_sbermarket_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.440.4511841640.ts)  
[typo_mage_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.440.4511841640.swift)  
[typo_mage_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.440.4511841640.kt)  
[typo_mage_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.440.4511841640.ts)  
[typo_plasma_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.440.4511841640.swift)  
[typo_plasma_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.440.4511841640.kt)  
[typo_plasma_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.440.4511841640.ts)  
[typo_ruler_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.440.4511841640.swift)  
[typo_ruler_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.440.4511841640.kt)  
[typo_ruler_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.440.4511841640.ts)  
[typo_sage_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.440.4511841640.swift)  
[typo_sage_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.440.4511841640.kt)  
[typo_sage_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.440.4511841640.ts)  
[typo_sbermarket_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.440.4511841640.swift)  
[typo_sbermarket_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.440.4511841640.kt)  
[typo_sbermarket_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.440.4511841640.ts)  
[typo_soulmate_ios-swift--canary.440.4511841640.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.440.4511841640.swift)  
[typo_soulmate_kotlin--canary.440.4511841640.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.440.4511841640.kt)  
[typo_soulmate_react-native--canary.440.4511841640.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.440.4511841640.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-440/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-440/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-440/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-440/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-440/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-440/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-440/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by kayman233 — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,43 +134,23 @@ git commit -m "fix(plasma-web): Fix component Y"
...
-
--   если Docker не установлен, установите его;
--   убедитесь, что докер запущен;
--   необходимо установить все пакеты в монорепозитории, для этого в корне монорепозитория выполните команды `npm ci`, а потом `npx lerna bootstrap`;
--   запустите прогон тестов, в корне монорепозитория выполните команду `npm run cy:ui:run-ct` (для остальных пакетов команды будут соответствующими);
--   обновите скриншоты, если это необходимо, для этого выполните `npm run cy:ui:run-ct-update-diffs`;
--   для отладки тестов, используйте команду `npm run cy:ui:open-ct`
+-   Собрана документация и Storybook's.
 
+Обновления можно посмотреть на главной странице и на https://www.npmjs.com/
```
Comment (Yeti-or) [inline CONTRIBUTING.md:156]: вот это не совсем понял

Comment (neretin-trike) [reply CONTRIBUTING.md:156]: тут раньше была не рабочая минифицированная ссылка на нашу главную plasma.sberdives. Решил без указания адреса просто описать, где глянуть можно. Предполагается, что контрибьютеры и так знают

Comment (Yeti-or) [reply CONTRIBUTING.md:156]: ну кажется бессмысленным тогда, может убрать эту строчку

**Comment by Yeti-or** [comment]
и ешё такой момент, а может мы не будем грузить все метаданные про наши пакеты из npm ;

А прочитаем тупо из репки во время сборки сайта ? типа require('./plasma-ui/package.json')

а за датой можно сходить в релизы github ? https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-a-release-by-tag-name

**Comment by Yeti-or** [comment]
и ещё мы не хотим прописать `font-display: swap` ?? https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display

File: CONTRIBUTING.md (hunk 1 above)
Comment (neretin-trike) [reply CONTRIBUTING.md:156]: оке

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-440/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-440/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Pull request merged**